crates/chain/src/tx_graph.rs

```diff
@@ -687,8 +701,20 @@ impl<A: Anchor> TxGraph<A> {
 
     /// Get the position of the transaction in `chain` with tip `chain_tip`.
     ///
-    /// If the given transaction of `txid` does not exist in the chain of `chain_tip`, `None` is
-    /// returned.
+    /// If the given `txid` does not exist in the chain of `chain_tip`, and
+    /// we believe that the transaction is not in mempool anymore, `None` is returned.
+    ///
+    /// We determine whether the transaction `tx` is in the mempool or not by looking at
+    /// tx's conflicting transactions, and tx's ancestors' conflicting transactions;
```

Comment:
```suggestion
    /// `tx`'s conflicting transactions, and `tx`'s ancestors' conflicting transactions;
```

Consistently mention `tx`